scripts/patch_version.py

```diff
@@ -33,14 +33,14 @@ def patch_version_lines(lines, build_number):
         yield line
 
 
-def patch_file(fname, build_number):
```

Comment:
this is some CI only script, not sure we need types here.

Reply:
I don't think it costs much to have the types that are easy though, and having them enables at least some checking from MyPy.

Reply:
what's the point of return only type annotation on a file only used on CI? You are changing script that has NO types at all, to a script with VERY PARTIAL types, to me this is worse than the prior state. Please revert this part.

Reply:
Fixed.